dspace-jspui/src/main/java/org/dspace/app/webui/servlet/VersionHistoryServlet.java

```diff
@@ -62,6 +63,11 @@ protected void doDSGet(Context context, HttpServletRequest request,
         	throw new IllegalArgumentException("Item is null");
         }
         
+        Boolean show_submitter = new DSpace()
+                .getConfigurationService()
+                .getPropertyAsType("versioning.item.history.include.submitter",
+                        new Boolean(false));
```

Comment:
This should just use `getBooleanProperty()` method of the ConfigurationService. Not sure why it needs to use `getPropertyAsType`?